crates/compiler/src/parser/generated.rs

```diff
@@ -2,6 +2,8 @@
 #![allow(clippy)]
 #![cfg_attr(rustfmt, rustfmt_skip)]
 
+//! Contains the modules that were generated by ANTLR.

```

Comment:
What does ! mean at the start of a comment?

Reply:
Same as in the distinction between #[attribute] and #![attribute]: it means "apply this to the item that opened this scope", which in this case is the mod. This makes the comments doc comments for the module